definitions/ext-solarwinds_network/golden_metrics.yml

```diff
@@ -0,0 +1,41 @@
+cpuPercent:
+  title: CPU percent
+  unit: PERCENTAGE
+  query:
+      select: latest(npm.avgCPULoad)
+      from: solarwinds_cpumem

```

Comment:
there are 2 golden metrics defined exactly the same.
Remove one and the validators will pass.

Reply:
Fixed